RELEASE.md

```diff
@@ -0,0 +1,37 @@
+# Release Process
+
+Pack follows a 6 week release cadence, composed of 3 phases:
+  - [Development](#development)
+  - [Feature Complete](#feature-complete)
+  - [Release Finalization](#release-finalization)
+
+## Development
+Our development flow is detailed in [Development](DEVELOPMENT.md)
+
+## Feature Complete
+### Process
+5 business days prior to a scheduled release, we enter `feature complete`. A release branch (in the form `release/<VERSION>`) is created, and used for `UAT` (User Acceptance testing).
+
+During this period, relevant changes may be merged into the release branch, based on assessment by the [CCB](#ccb) of the impact, effort and risk of including the changes. Any other change may get merged into `main` through the normal process, and will make it into the next release.
+
+### Roles
+#### CCB
+The `CCB` (Change Control Board) are the `pack` [maintainers][maintainers].
+
+#### Release Manager
+One of the [maintainers][maintainers] is designated as the release manager. They communicate the release status to the working group meetings, schedule additional meetings with the [CCB](#ccb) as needed, and finalize the release. They also take care of whatever release needs may arise.
+
+## Release Finalization
+The [release manager](#release-manager) will:
+- Create a [github release][release], containing the **artifacts**, **release notes**, and a **migration guide** (if necessary), documenting breaking changes, and providing actions to migrate from prior versions.
+- Tag the release branch as `v<version>`
+- Merge the release branch into `main`
+- Send out release notifications on
```

Comment:
```suggestion
- Send out release notifications, if deemed necessary, on
```

This has naturally changed so that it's rolled up in a monthly update for the newsletter and only tweeted about when deemed necessary (high-profile feature, urgent fix, etc).

Reply:
Resolved in becdf38e75f60d7a9553381dfd4b8f9bb928c21f